src/buttons/input.coffee

```diff
@@ -94,12 +94,20 @@ class InputPopover extends Popover
             </select>
           </td>
         </tr>
+        <tr>
+          <td class="field-name">
+            <label>Columns:</label>
+          </td>
+          <td>
+            <input class="simditor-input-width" type="number" min="0" />
```

Comment:
Min not should be 1 like in the textarea config?

Reply:
If it's 0 then it's falsy and removes the attribute as a whole